### PR TITLE
ASimAuditEventCiscoMeraki.yaml-3

### PR DESCRIPTION
**Pull request by Alekhya0824**
Required items, please complete
   
   Change(s):
   - Created a new file for Syslog and added Syslog parser in it. In Cisco Meraki file removed Syslog parser and in ASimAuditEvent added Syslog Parser name  and same changes have done for vim as well.

   Reason for Change(s):
   - To Split the Cisco Meraki and Syslog parser separately

   Version Updated:
   - yes

   Testing Completed:
   - yes

   Checked that the validations are passing and have addressed any issues that are present:
   - yes

### CODE REVIEW

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Please investigate the validation failures. Thanks

File: Parsers/ASimAuditEvent/Parsers/ASimAuditEvent.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Parser:
```
Comment (vakohl) [inline Parsers/ASimAuditEvent/Parsers/ASimAuditEvent.yaml:1]: why there are KQL tests related changes?

**Review by vakohl — CHANGES_REQUESTED** [review]
review done

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Please see above open comments and act on them. Thanks

File: Parsers/ASimAuditEvent/Parsers/ASimAuditEvent.yaml (hunk 1 above)
Comment (Alekhya0824) [reply Parsers/ASimAuditEvent/Parsers/ASimAuditEvent.yaml:1]: i'm updating the schema table so kql tests is changing

Comment (vakohl) [reply Parsers/ASimAuditEvent/Parsers/ASimAuditEvent.yaml:1]: which schema table alekhya?

**Review by vakohl — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Review by vakohl — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**